ERCS/erc-7222.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Count Contract
+description: Designed to manage and keep track of a numerical count.
+author: Yogesh Sahu (@yogesh8287)
+discussions-to: https://ethereum-magicians.org/t/eip-social-recovery-interface/14494
```

Comment:
```suggestion
discussions-to: [to be created]
```
Please create a discussions topic on https://ethereum-magicians.org and add the link here.